install_config/topics/configuring_for_vsphere.adoc

```diff
@@ -186,3 +179,7 @@ app-2.example.com openshift_node_group_name="node-config-compute"
 default container image registry, specify the credentials for that account. See
 xref:../install_config/configuring_red_hat_registry.html#install-config-configuring-red-hat-registry[Accessing and Configuring the Red Hat Registry].
 
+[NOTE]
+====
+link:https://access.redhat.com/documentation/en-us/reference_architectures/2019/html-single/deploying_and_managing_openshift_3.11_on_a_vmware_software-defined_data_center/index#deploying_a_vsphere_vm_environment_optional[Deploying a vSphere VM environment] is not officially supported by Red Hat, but it can be configured.
```

Comment:
We don't usually doc things that aren't supported, so removing the option is the right thing to do, but I think that Katherine or Eric should weigh in on the link.

Reply:
After reading through the Bugzilla, I'm good with removing this as well. Otherwise, Maria Bracho is the PM for the VMware. She should at least be made aware of the change.

Reply:
I missed that Bugzilla had logged me out, but with the rest of the conversation, it does make sense.

Reply:
@katherinedube just so we're on the same page, do suggest we also remove the link? Or is it OK to link out to the reference architecture article?

Reply:
I think the link is fine. As long as we remove the unsupported vm deployment section.